### PR TITLE
Fix typo in readme

### PR DESCRIPTION
**Pull request by rickhanlonii**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -516,7 +516,7 @@ something about it.
 When alex launched, it got some traction on [twitter][] and [producthunt][].
 Then there was a [lot][tnw] [of][dailydot] [press][vice] [coverage][bustle].
 
-There there was some backlash 🤷
+Then there was some backlash 🤷
```
Comment (rickhanlonii) [inline readme.md]:
```suggestion
```

Would recommend just removing this sentence 👍

Comment (wooorm) [reply readme.md]: I can’t change your fork, can you submit that suggestion?

File: readme.md [hunk 2]
```diff
@@ -516,7 +516,6 @@ something about it.
 When alex launched, it got some traction on [twitter][] and [producthunt][].
 Then there was a [lot][tnw] [of][dailydot] [press][vice] [coverage][bustle].
 
-There there was some backlash 🤷
```
Comment (rickhanlonii) [inline readme.md]:
```suggestion
```

**Comment by rickhanlonii** [comment]
@wooorm done

**Comment by wooorm** [comment]
Sweet, thanks!

**Pull request merged**